stock_landed_costs_security/views/stock_landed_costs_view.xml

```diff
@@ -13,12 +13,6 @@
             </button>
         </field>
     </record>
-    <record id="stock_landed_costs.account_view_move_form_inherited" model="ir.ui.view">
-        <field
-            name="groups_id"
-            eval="[(4, ref('stock_landed_costs_security.group_stock_landed_costs_security_user'))]"
```

Comment:
You must put this group on each of the fields of the view.

Reply:
I have checked and the fields that are invisibile should i add the group also? https://github.com/odoo/odoo/blob/16.0/addons/stock_landed_costs/views/account_move_views.xml#L3
I tried on my local but adding the groups in the others fields that are invisible i got error message.

Reply:
As said, you now need to put the groups attribute on each of the elements of the view:

![image](https://github.com/OCA/stock-logistics-workflow/assets/7165771/90d4a77a-b9c7-4d0a-8dbd-2e26ff17b3be)

Reply:
Done! Thanks